jib-core/src/test/java/com/google/cloud/tools/jib/docker/DockerClientTest.java

```diff
@@ -176,11 +181,37 @@ public void testTag() throws InterruptedException, IOException, InvalidImageRefe
               Assert.assertEquals(Arrays.asList("tag", "original", "new"), subcommand);
               return mockProcessBuilder;
             });
-    Mockito.when(mockProcess.waitFor()).thenReturn(0);
+    when(mockProcess.waitFor()).thenReturn(0);
 
     testDockerClient.tag(ImageReference.of(null, "original", null), ImageReference.parse("new"));
   }
 
+  @Test
+  public void testDefaultClient() {
+    DockerClient dc = DockerClient.newClient();
+    Assert.assertEquals(
+        dc.getProcessBuilderFactory().apply(Collections.emptyList()).command(),
+        Collections.singletonList("docker"));
```

Comment:
In our repo, we reverse the order so that `assertEquals(expected, actual)`.

Reply:
addressed